libs/dataset-explorer/src/lib/FeatureBalanceMeasuresChart.tsx

```diff
@@ -0,0 +1,276 @@
+                            width: "100%"
+                          }
+                        }
+                      },
+                      dropdown: {
+                        width: 150
+                      }
+                    }}
+                    id="labelDropdown"
+                    options={labelOptions}
+                    selectedKey={this.state.selectedLabelIndex}
+                    onChange={this.setSelectedLabel}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+
+            {/* Renders the Feature dropdown and its heading */}
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "s1" }}>
+                <Stack.Item>
+                  <Text variant="mediumPlus" className={styles.boldText}>
+                    {measuresLocalization.FeaturePicker}
+                  </Text>
+                </Stack.Item>
+                <Stack.Item>
+                  <Dropdown
+                    styles={{
+                      callout: {
+                        selectors: {
+                          ".ms-Button-flexContainer": {
+                            width: "100%"
+                          }
+                        }
+                      },
+                      dropdown: {
+                        width: 150
+                      }
+                    }}
+                    id="featureDropdown"
+                    options={featureOptions}
+                    selectedKey={this.state.selectedFeatureIndex}
+                    onChange={this.setSelectedFeature}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+
+            {/* Renders the Measure dropdown and its heading */}
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "s1" }}>
+                <Stack.Item>
+                  <Text variant="mediumPlus" className={styles.boldText}>
+                    {measuresLocalization.MeasurePicker}
+                  </Text>
+                </Stack.Item>
+                <Stack.Item>
+                  <Dropdown
+                    styles={{
+                      callout: {
+                        selectors: {
+                          ".ms-Button-flexContainer": {
+                            width: "100%"
+                          }
+                        }
+                      },
+                      dropdown: {
+                        width: 200
+                      }
+                    }}
+                    id="measureDropdown"
+                    options={measureOptions}
+                    selectedKey={this.state.selectedMeasureIndex}
+                    onChange={this.setSelectedMeasure}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+
+            {/* Renders the description of what user chose from dropdowns */}
+            <Stack.Item>
```

Comment:
nit: can you refactor this out into a separate description component?  we try to keep files under 300 lines or even smaller if possible.  I think this could be its own class.